ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -349,12 +349,13 @@ let assert_host_has_iommu ~__context ~host =
   if List.assoc "iommu" chipset_info <> "true" then
     raise (Api_errors.Server_error (Api_errors.vm_requires_iommu, [Ref.string_of host]))
 
-(* Check if there are vgpus not allocated, since we may check the mem/vgpu after reservation *)
+(* Check if there are vgpus not allocated, since we may check the mem/vgpu after reservation 
+   Assumption: a VM can have only one vGPU *)
```

Comment:
Let's move this Assumption comment below, at line 518, where you call `assert_gpus_available`. This function does not really care how many VGPUs there are

Reply:
Sure.